internal/cloudrunci/example_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudrunci_test
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+)
+
+func init() {
+	os.Setenv("BUILD_ID", "12345")
+}
+
+// ExampleService_Build shows how to build once, deploy many.
+func ExampleService_Build() {
```

Comment:
Probably OK if we don't have these, given there will be other usages. But, I'm not opposed.

Reply:
This dates to plans for standalone, where I wanted to illustrate some more how-to guidance. Since no objections, would like to leave it in for now but if maintenance is an issue we can remove.